data_structures/dict_text_analysis_test.py

```diff
@@ -0,0 +1,25 @@
+# test cases dictionary example to count words in a file.
+
+
+import pytest
+from dict_text_analysis import text_analysis
+import os
+
+
+@pytest.mark.parametrize(
+    "file_name, result",
+    [
+        (
+            "/test1.txt",
```

Comment:
Remove `/` here and below.

`os.path.join` adds the path delimiters on its own based on what OS you are on.

Reply:
file name is corrected and os.path.join is used